docs/Language Feature Status.md

```diff
@@ -0,0 +1,31 @@
+# Language Feature Status
+
+## C# 7.0 and VB 15
+
+| Feature | Branch | State | Owners |
+| ------- | ------ | ----- | ------ | ----- |
+| Async Main | none  | Feature Specification | [tyoverby](https://github.com/tyoverby), [agocke](https://github.com/agocke) |
```

Comment:
You should probably use github-user links like @agocke here. Though it would be useful for mentioned people to be notified of mentions in the doc, I suspect github doesn't do that.